assets/docs/configuration/targets/http-full-example.hcl

```diff
@@ -62,5 +62,27 @@ target {
 
     # Optional path to the file containing template which is used to build HTTP request based on a batch of input data
     template_file              = "myTemplate.file"
+
+    # 2 invalid + 1 setup error rules
+    response_rules {
+      # This one is a match when... 
+      invalid {
```

Comment:
Can we configure rules for retryable errors, or is that just always the default case?